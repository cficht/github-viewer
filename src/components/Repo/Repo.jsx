import React from 'react';
import PropTypes from 'prop-types';

const Repo = ({ name, html_url }) => (
  <li>
    <h2>{name}</h2>
    <p>{html_url}</p>
  </li>
);

Repo.propTypes = {
  name: PropTypes.string.isRequired,
  html_url: PropTypes.string.isRequired
};

export default Repo;
