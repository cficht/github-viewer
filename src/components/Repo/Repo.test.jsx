import React from 'react';
import { shallow } from 'enzyme';
import Repo from './Repo';

describe('Repo component', () => {
  it('renders Repo', () => {
    const wrapper = shallow(<Repo 
      name="about-me"
      html_url="https://github.com/cficht/about-me"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
