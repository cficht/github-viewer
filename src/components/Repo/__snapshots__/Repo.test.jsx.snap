// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo component renders Repo 1`] = `
<li>
  <p>
    about-me
  </p>
  <p>
    https://github.com/cficht/about-me
  </p>
</li>
`;
